controller/test/blower_fsm/blower_fsm_test.cpp

```diff
@@ -16,23 +16,37 @@ limitations under the License.
 #include "blower_fsm.h"
 
 #include "hal.h"
+#include "gmock/gmock-matchers.h"
 #include "gtest/gtest.h"
 
+namespace {
+
 class BlowerFsmTest : public ::testing::Test {
 public:
   BlowerFsmTest() = default;
   void SetUp() override { blower_fsm_init(); }
 };
 
+// One could also accopmlish this with gmock matchers, but wow is this easier.
+//
+// Has to be a macro and not a function, otherwise gunit can't tell us exactly
+// which expectation failed.
+#define EXPECT_STATE_IS(state, pressure_cm_h2o, valve_state)                   \
+  do {                                                                         \
+    BlowerSystemState s = (state);                                             \
+    EXPECT_FLOAT_EQ(s.pressure.cmH2O(), (pressure_cm_h2o));                    \
+    EXPECT_EQ(s.expire_valve_state, ValveState::valve_state);                  \
+  } while (0)
+
 TEST_F(BlowerFsmTest, InitiallyOff) {
   VentParams p = VentParams_init_zero;
-  EXPECT_EQ(blower_fsm_get_setpoint(p), kPa(0));
+  EXPECT_STATE_IS(blower_fsm_desired_state(p), 0, OPEN);
```

Comment:
Please annotate the "0" and other similar arguments below as `/*pressure_cm_h2o=*/` for readability